### PR TITLE
BUGFIX: TMA-1505 import gooddata certificate when run smoke test

### PR DESCRIPTION
**Pull request by phong-nguyen-duy**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1549/ : SUCCESS in 13s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/327/ : SUCCESS in 59s

File: .travis.yml [hunk 1]
```diff
@@ -122,7 +122,11 @@ jobs:
   - &bricks-smoke-tests
     name: staging1 - bricks smoke tests
     stage: after-merge
-    script: bundle exec rake -f lcm.rake test:smoke
+    script:
+      - sudo keytool -importcert -alias gooddata-2008 -file "./data/2008.crt" -keystore $JAVA_HOME/jre/lib/security/cacerts -trustcacerts -storepass 'changeit' -noprompt
+      - sudo keytool -importcert -alias gooddata-int -file "./data/new_ca.cer" -keystore $JAVA_HOME/jre/lib/security/cacerts -trustcacerts -storepass 'changeit' -noprompt
```
Comment (juras-gdc) [inline .travis.yml]: still wondering, do we really need other certificates then the "new_ca" here?
these tests connect to "staging" only, therefore prodgdc and obsoleted gdc-2008 should not be needed, right ?

Comment (phong-nguyen-duy) [reply .travis.yml]: This configuration is reused for staging2 and staging3 as well.

Comment (juras-gdc) [reply .travis.yml]: All stagings use the latest gdc certificate. I doubt the old one use useful for anything, frankly I have no idea what service could rely on obsoleted certificate. Regarding prod-certificate I'm not aware any tests is executed against production environment neither perf cluster.

IMO the "new_ca" should be the only certificate used in whole configuration file (btw. checked these test run locally with new_ca only).

But this is not crucial issue right now, we need to remember to revalidate usage of certs later ;-)

**Pull request merged**